configs/projects/pdk-runtime.rb

```diff
@@ -186,11 +186,13 @@
   if proj.respond_to?(:additional_rubies)
     proj.additional_rubies.keys.each do |rubyver|
       proj.component "ruby-#{rubyver}"
-    end
 
-    proj.component "ruby-multi-augeas" unless platform.is_windows?
-    proj.component "ruby-multi-selinux" if platform.name =~ /^el-(5|6|7)-.*/ || platform.is_fedora?
-    proj.component "ruby-multi-stomp"
+      ruby_minor = rubyver.split('.')[0,2].join('.')
+
+      proj.component "ruby-#{ruby_minor}-augeas" unless platform.is_windows?
+      proj.component "ruby-#{ruby_minor}-selinux" if platform.name =~ /^el-(5|6|7)-.*/ || platform.is_fedora?
```

Comment:
This needs to be updated to use `platform.is_el?` like the way it's done for the main ruby.

Reply:
Updated